src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -28,9 +28,12 @@ export default class GuideDrawer extends React.Component {
         <div className="assistant-drawer-title">
           {this.props.guide.steps[this.props.step - 1].title}
         </div>
-        <div className="assistant-drawer-message">
-          {this.props.guide.steps[this.props.step - 1].message}
-        </div>
+        <div
+          className="assistant-drawer-message"
+          dangerouslySetInnerHTML={{
```

Comment:
@lynnagara this is ok for stuff we hardcode right?

Reply:
Yeah I think so. I assume each message will need to have very different markup / styles, hence the need to define it separately for each one?

Reply:
Right now I'm not anticipating anything more complicated than links. But yeah each message is custom.